static/src/stylesheets/module/content-garnett/live-blog/_live-blog.head.scss

```diff
@@ -813,13 +815,16 @@ $pillars: (
         }
 
         .tonal__standfirst {
-            background: lighten(map-get($palette, pillarColor), 10);
+            background: lighten(map-get($palette, pillarColor), 5);
```

Comment:
I checked in SASS, it is using HSL as is standard in CSS. Your site is probably using HSB (dick move), which is [very different](https://en.wikipedia.org/wiki/HSL_and_HSV):
> In HSL, saturation is also unchanged by tinting with white, and only mixtures with both black and white—called tones—have saturation less than 1. In HSV, tinting alone reduces saturation.

Reply:
Thanks Regis, 5% Lighten gets the affect we're looking for.